styles/style.css

```diff
@@ -95,7 +97,25 @@ p {
 
 /* Buttons */
 
-  /* INSERT BUTTON STYLES HERE */
+  button{
```

Comment:
Instead of using the tag name, use a class name instead. The reason is that, some elements can be represented in different ways using different tags, but they can be the same in terms of styles. Example, `button` and `input[type=submit]` are two elements that are often used to display buttons.